custom_components/tuya_local/devices/dreamegg_nite_1.yaml

```diff
@@ -0,0 +1,305 @@
+          max: 1000
+        mapping:
+          - target_range:
+              min: 2700
+              max: 6500
+      - id: 5
+        name: rgbhsv
+        type: hex
+        optional: true
+        format:
+          - name: h
+            bytes: 2
+            range:
+              min: 0
+              max: 360
+          - name: s
+            bytes: 2
+            range:
+              min: 0
+              max: 1000
+          - name: v
+            bytes: 2
+            range:
+              min: 0
+              max: 1000
+
+  - entity: siren
+    name: Music
+    icon: "mdi:surround-sound"
+    dps:
+      - id: 8
+        name: switch
+        type: boolean
+      - id: 9
+        name: volume_level
+        type: integer
+        range:
+          min: 0
+          max: 100
+        mapping:
+          - scale: 100
+      - id: 10
+        name: tone
+        type: string
+        mapping:
+          - dps_val: "1"
+            value: Bird
+          - dps_val: "2"
+            value: Cricket
+          - dps_val: "3"
+            value: Campfire
+          - dps_val: "4"
+            value: Gentle Rain
+          - dps_val: "5"
+            value: Heavy Rain
+          - dps_val: "6"
+            value: Rainstorm
+          - dps_val: "7"
+            value: Brook
+          - dps_val: "8"
+            value: Gentle Surf
+          - dps_val: "9"
+            value: Sea Wave
+          - dps_val: "10"
+            value: Waterfall
+          - dps_val: "11"
+            value: Wind
+          - dps_val: "12"
+            value: Sushing
+          - dps_val: "13"
+            value: Womb Sound
+          - dps_val: "14"
+            value: Hair Dryer
+          - dps_val: "15"
+            value: Airplane
+          - dps_val: "16"
+            value: Cruise Ship
+          - dps_val: "17"
+            value: Train
+          - dps_val: "18"
+            value: Wind Chimes
+          - dps_val: "19"
+            value: White Noise
+          - dps_val: "20"
+            value: Pink Noise
+          - dps_val: "21"
+            value: Brown Noise
+          - dps_val: "22"
+            value: Gentle Fan
+          - dps_val: "23"
+            value: Box Fan
+          - dps_val: "24"
+            value: Oscillating Fan
+          - dps_val: "25"
+            value: Zen
+          - dps_val: "26"
+            value: Meditation
+          - dps_val: "27"
+            value: Lullaby
+          - dps_val: "28"
+            value: Little Star
+          - dps_val: "29"
+            value: Bedtime
+          - dps_val: "30"
+            value: Brahm's Lullaby
+          - dps_val: "31"
+            value: Music Box
+          - dps_val: "32"
+            value: Piano
+          - dps_val: "33"
+            value: Mozart
+          - dps_val: "34"
+            value: Morning
+      - id: 20
+        name: duration
+        type: integer
+        unit: min
+        range:
+          min: 0
+          max: 1440
+
+  - entity: select
```

Comment:
It is better not to duplicate entities just to work around limitations in the HA UI compared with a personal preference for how you want your UI to work. You can use a select helper if you really need a select here (though you likely need to manually input all the options for that).